test/interop/C/exportString/noArgsRetString.suppressif

```diff
@@ -0,0 +1,5 @@
+#
+# The Cray CC is a bit more strict about pointer conversions and does not have
+# a flag equivalent to "-Wno-pointer-sign".
```

Comment:
You might adjust the phrasing to say '... a flag equivalent to gcc's "-Wno-pointer-sign"', so it's obvious where someone curious about what that is would go to find out more.  (same for the other file also, of course)